CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+
+# Code of Conduct
+Please read our [Code of Conduct](https://github.com/stride-nyc/remote_retro/blob/master/CODE_OF_CONDUCT.md) before contributing.
+
+# First time contributing?
+Unsure where to begin contributing to Remote Retro? You can start by looking through [issues](https://github.com/stride-nyc/remote_retro/issues) that have the "beginner-friendly" or "help wanted" label.
+
+# Getting started
+* Before you start coding, please see the [Dev Environment Setup](https://github.com/stride-nyc/remote_retro#dev-environment-setup) and [Tests](https://github.com/stride-nyc/remote_retro#tests) sections of the Readme for information about getting set up and running the tests.
+* We require all contributors to sign a Contributor License Agreement before contributing. You will be prompted to electronically sign the individual CLA when you make a pull request.
+  - The individual CLA can be viewed [here](https://gist.github.com/qlaire/cfcc3a2b9ab28f3c97716e4040afe578)
+  - The corporate CLA can be viewed [here](https://gist.github.com/qlaire/c6b31ad2ba9489e75f6c810be466e7c6)
+* Please see the [Project Management](https://github.com/stride-nyc/remote_retro#project-management) section to see how to view Remote Retro's current feature pipeline.
+
+Once you're ready to start coding:
+
+1. Create your own fork of the code
+2. Do the changes in your fork
+3. If you like the change and think the project could use it:
+    * Be sure you have followed the code style for the project.
+    * Make a pull request, and be sure to fill out the information in the template.
+
+
+# How to report a bug
+### Security issues
+If you find a security vulnerability, do NOT open an issue. Email XXXX instead.
+
+In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+* Can I access something that's not mine, or something I shouldn't have access to?
+* Can I disable something for other people?
+
+### For non-security issues
+Please open an issue and use the template provided. Please add the "Bug" label.
+
+# How to suggest a feature or enhancement
+Before suggesting a feature, please read through the [Roadmap to MVP](https://github.com/stride-nyc/remote_retro#roadmap-to-mvp) and check the backlog to make sure that the story doesn't exist already. Once you're ready to suggest the feature or enhancement, open an issue and use the provided template. Please also add the "feature request" label.
+
+# Code review process
+After you submit your pull request, it will be reviewed within 10 business days and we will give you feedback on any needed changes. After feedback has been given we expect responses within two weeks. After two weeks we may close the pull request if it isn't showing any activity.
+
+# Code Style
+ESLint is used for checking JavaScript code style. It is also useful for avoiding some errors. Please use indentation of 2 spaces, and no semicolons. Please use ES2015.
```

Comment:
Maybe add `(ES6)` after "ES2015" to avoid possible confusion?

Reply:
on it.